### ISSUE TITLE
Build failure on DL4J

### ISSUE THREAD

**Issue by huitseeker** [deeplearning4j/deeplearning4j#4115]
#### Issue Description

```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/huitseeker/DL4J/deeplearning4j/deeplearning4j-zoo/src/main/java/org/deeplearning4j/zoo/model/GoogLeNet.java:[176,40] cannot find symbol
  symbol:   variable InputType
  location: class org.deeplearning4j.zoo.model.GoogLeNet
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] DeepLearning4j ..................................... SUCCESS [  2.127 s]
[INFO] deeplearning4j-nearestneighbors-parent ............. SUCCESS [  0.185 s]
[INFO] nearestneighbor-core ............................... SUCCESS [  6.824 s]
[INFO] deeplearning4j-nn .................................. SUCCESS [ 10.407 s]
[INFO] deeplearning4j-modelimport ......................... SUCCESS [  5.669 s]
[INFO] deeplearning4j-ui-parent ........................... SUCCESS [  0.141 s]
[INFO] deeplearning4j-ui-components ....................... SUCCESS [  4.004 s]
[INFO] deeplearning4j-core ................................ SUCCESS [  9.189 s]
[INFO] deeplearning4j-ui-model ............................ SUCCESS [  6.298 s]
[INFO] deeplearning4j-play ................................ SUCCESS [  8.949 s] 
[INFO] DeepLearning4j-scaleout-parent ..................... SUCCESS [  0.085 s] 
[INFO] org.deeplearning4j.spark ........................... SUCCESS [  2.351 s] 
[INFO] dl4j-spark ......................................... SUCCESS [ 21.994 s] 
[INFO] Dl4j Streaming ..................................... SUCCESS [  6.267 s] 
[INFO] DeepLearning4j-AWS ................................. SUCCESS [  2.543 s] 
[INFO] deeplearning4j-nlp-parent .......................... SUCCESS [  0.078 s] 
[INFO] deeplearning4j-nlp ................................. SUCCESS [  8.199 s] 
[INFO] dl4j-spark-nlp ..................................... SUCCESS [ 10.673 s] 
[INFO] deeplearning4j-parallel-wrapper .................... SUCCESS [  4.291 s] 
[INFO] dl4j-spark-parameterserver ......................... SUCCESS [ 11.317 s] 
[INFO] dl4j-spark-nlp-java8 ............................... SUCCESS [ 10.248 s] 
[INFO] dl4j-spark-ml ...................................... SUCCESS [ 15.238 s] 
[INFO] deeplearning4j-scaleout-parallelwrapper-parameter-server SUCCESS [  3.781 s]
[INFO] deeplearning4j-ui-resources ........................ SUCCESS [  0.801 s] 
[INFO] deeplearning4j-ui .................................. SUCCESS [  3.810 s] 
[INFO] deeplearning4j-ui-remote-iterationlisteners ........ SUCCESS [  2.127 s] 
[INFO] deeplearning4j-graph ............................... SUCCESS [  3.224 s] 
[INFO] deeplearning4j-nlp-uima ............................ SUCCESS [  4.347 s] 
[INFO] deeplearning4j-nlp-korean .......................... SUCCESS [  2.758 s] 
[INFO] deeplearning4j-nlp-japanese ........................ SUCCESS [  4.016 s] 
[INFO] deeplearning4j-nlp-chinese ......................... SUCCESS [  5.672 s] 
[INFO] deeplearning4j-zoo ................................. FAILURE [  1.998 s] 
[INFO] deeplearning4j-nearestneighbors-model .............. SKIPPED             
[INFO] deeplearning4j-nearestneighbors-client ............. SKIPPED             
[INFO] deeplearning4j-nearestneighbor-server_2.11 ......... SKIPPED             
[INFO] ------------------------------------------------------------------------ 
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------ 
[INFO] Total time: 03:00 min            
[INFO] Finished at: 2017-09-26T13:25:50-07:00                                   
[INFO] Final Memory: 265M/3102M         
[INFO] ------------------------------------------------------------------------ 
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project deeplearning4j-zoo: Compilation failure
[ERROR] /home/huitseeker/DL4J/deeplearning4j/deeplearning4j-zoo/src/main/java/org/deeplearning4j/zoo/model/GoogLeNet.java:[176,40] cannot find symbol
[ERROR]   symbol:   variable InputType  
[ERROR]   location: class org.deeplearning4j.zoo.model.GoogLeNet                
[ERROR]             
[ERROR] -> [Help 1] 
```

Full log: https://gist.github.com/a0aaf4b5eddbe3294b90895f8cadfb04

Full versions:
```
huitseeker@pegasus➜/home/huitseeker/DL4J» for i in libnd4j nd4j datavec deeplearning4j; do pushd $i; git rev-parse HEAD; popd -q; done
/home/huitseeker/DL4J/libnd4j /home/huitseeker/DL4J /home/huitseeker
3c8dc2d566da960147c75290c2edca8941aa2e0b
/home/huitseeker/DL4J/nd4j /home/huitseeker/DL4J /home/huitseeker
96e76d8a21d4a81e336e4db30212f5ba78c3a188
/home/huitseeker/DL4J/datavec /home/huitseeker/DL4J /home/huitseeker
4f044e8962ed71fae5eddd28c3987c5af22de3d4
/home/huitseeker/DL4J/deeplearning4j /home/huitseeker/DL4J /home/huitseeker
6b3fab4b6319d6c18a0e4e2c982589b30465cfa3
```

Most certainly a result of #3955 (last line of the PR diff).

**Comment by crockpotveggies** [comment]
Something changed in the last two weeks this PR was valid. I'll fix it

**Comment by huitseeker** [comment]
Fixed in #4116
